codegen/scala-gen/src/main/scala/com/akkaserverless/codegen/scalasdk/impl/ValueEntitySourceGenerator.scala

```diff
@@ -60,8 +60,8 @@ object ValueEntitySourceGenerator {
     val stateType = valueEntity.state.fqn.name
     val packageName = valueEntity.fqn.parent.scalaPackage
     val className = valueEntity.fqn.name
-
-    val imports = generateCommandImports(
+    val abstractEntityName = valueEntity.abstractEntityName
+    implicit val imports = generateCommandImports(
```

Comment:
note that in generateImplementationSkeleton it's using generateImports and not generateCommandImports. There it's also defining `packageImports`. I don't know if it should be the same here or not. @raboof can maybe clarify?

Reply:
I don't know how this works yet, I'll have a look.

Reply:
Yes, that'll be needed here as well to fix https://github.com/lightbend/akkaserverless-java-sdk/pull/453/files#r713907871